tests/integration/user_management/claims_repository/test_deletion.py

```diff
@@ -81,8 +81,9 @@ async def test_deletion_handler(
         )
         await event_subscriber.run(forever=False)
         records = caplog.records
-        assert len(records) == 2, records
+        assert len(records) == 4, records
```

Comment:
The two extra logs are from the logs added to the kafka providers during publish and consume, right?

Reply:
Exactly.

Reply:
Made this now more obvious in the test.